### ISSUE TITLE
Slider input has no accessible name

### ISSUE THREAD

**Issue by kirsty-hames** [adaptlearning/adapt-contrib-confidenceSlider#99]
### Subject of the issue
[Mozilla ARIA slider role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/slider_role), _**An accessible name is required**. If the range's role is applied to an HTML `<input>` element (or `<meter>` or `<progress>` element), the accessible name can come from the associated `<label>`. Otherwise use `aria-labelledby` if a visible label is present or `aria-label` if a visible label is not present._

### Expected behaviour
The slider `<input type="range">` should have an appropriate `aria-label` or `aria-labelledby` property set.

Same issue raised for [Slider](https://github.com/adaptlearning/adapt-contrib-slider/issues/189).
